doc/chapters/gateways/index.rst

```diff
@@ -302,12 +302,39 @@ An example configuration can look like this::
 Network gateway
 ===============
 
-The Network Gateway is used to provide Internet access through a container-specific firewall.
+The Network Gateway is used to setup network connection and configure which traffic is allowed and not.
 
 Configuration
 -------------
+Types: "INCOMING" and "OUTGOING"
+Targets: "ACCEPT", "DROP" and "REJECT"
+host: hostname or ip-address with netmask
+default: target to choose when not matching any other rule
 
-T.B.D.
+Example of network gateway config::
```

Comment:
This is interesting imho. To begin with we could be lazy and just let the rules wind up in the order they are added, there is the possibility to add rules in different ordering from each rule (ie, inserting, prepending, appending, etc), which might get confusing... we could add a match/add value which marks the rule for appending or prepending perhaps? The inserting could become really hairy otherwise.